tests/src/integration/masp.rs

```diff
@@ -1014,7 +1014,7 @@ fn masp_txs_and_queries() -> Result<()> {
                 "--node",
                 validator_one_rpc,
             ],
-            Response::Ok("Transaction is valid"),
+            Response::Ok("Transaction was successfully applied"),
```

Comment:
```suggestion
            Response::Ok(TX_APPLIED_SUCCESS),
```
maybe we should make this a const somewhere? next time we change this, if any, will result in a shorter diff

Reply:
good call, will add it to e2e tests too

Reply:
I've added bunch of strings in https://github.com/anoma/namada/pull/2276/files/f167f2b5b8396acf148fcc9cf992daf89f3b48a7..d09bc55133bfb8dddbadb9b9f5aa6ae2739718d6